lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_clear_intermediate_tags.rb

```diff
@@ -5,18 +5,20 @@ def self.run(params)
         UI.message("Deleting tags for version: #{params[:version]}")
 
         require_relative '../../helper/git_helper.rb'
-        Fastlane::Helper::Ios::GitHelper.delete_tags(params[:version])
 
-        # Cleanup local tags and refetch them to make sure we're up-to-date with remote
-        local_tags = Fastlane::Helper::GitHelper.list_local_tags()
-        Fastlane::Helper::GitHelper.delete_tags(local_tags, delete_on_remote: true)
-        Fastlane::Helper::GitHelper.fetch_all_tags()
+        # Delete the current version's tag (if it exists)
+        Fastlane::Helper::GitHelper.delete_tags(params[:version])
```

Comment:
Hmmm... this was introduced during the refactoring, but I'm not sure why we need it 🤔 
I don't expect a tag to be present at this time and, if it is, I don't think we want to delete it silently.

Reply:
Yeah, good question – so my thinking on this is that if we're re-running this, it's likely due to a failed finalization, in which case we want it to be idempotent. Though what you say about not deleting it silently seems good from a "confirm destructive operations" standpoint (even if we'll re-create the tag right away).

WDYT about keeping it, but asking for confirmation?

Reply:
With the new flow, the final tag is created remotely when the release is published. But also with the old flow (which we still have on Woo and Simplenote) creating the tag is the last step, so we should never need it. 

Tbh, I think this was just a mistake during the refactoring, but if you really want to keep it for any reasons, I agree that we'd need at least a confirmation because it would be a really weird situation.
Also, in this case, I think it makes more sense to delete the tag remotely as well... otherwise we are just going to defer the failure.

Reply:
I still think it might be useful, but for the purposes of this PR (which is to fix a bug), it doesn't need to be here. We can always make the finalization idempotent another time. 

Removed in 629cc8c19e78db695cce8cf3297ad0fb5c9dfa10